### PR TITLE
Feature/212 move layer inspector to bottom center

### PR DESCRIPTION
**Pull request by muganwas**

### CODE REVIEW

File: src/components/play/PlayUI.js [hunk 1]
```diff
@@ -47,16 +47,19 @@ class PlayUI extends Component {
     async componentDidMount() {
         // register this component with parent so we can do some instant updates bypassing redux for speed
         this.props.childRef(this)
-
         await this.createRound()
-        window.addEventListener('resize', this.onWindowResizeThrottled)
-        window.addEventListener('keypress', this.onKeypress)
-        window.addEventListener('dblclick', () => this.onMuteToggle(this.props))
+        if (window) {
```
Comment (choshun) [inline src/components/play/PlayUI.js:51]: This should always be defined

File: src/components/play/PlayUI.js [hunk 2]
```diff
@@ -88,9 +98,15 @@ class PlayUI extends Component {
         this.draw()
     }
 
-    async componentDidUpdate() {
-        // console.log('componentDidUpdate()', this.round, this.props.round)
-        // console.time('componentDidUpdate')
+    async componentDidUpdate(prevProps) {
+        console.log('componentDidUpdate()', this.round, this.props.round)
```
Comment (choshun) [inline src/components/play/PlayUI.js:102]: console

File: src/components/play/layer-settings/LayerInstrumentAlt.js [hunk 3]
```diff
@@ -0,0 +1,159 @@
+import React, { useContext } from 'react'
```
Comment (choshun) [inline src/components/play/layer-settings/LayerInstrumentAlt.js:1]: If there's a new component "alt", did you delete the new one?

Is there a reason you didn't change the old one?

File: src/components/play/layer-settings/LayerInstrumentAlt.js [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+            firebase.updateLayer(roundId, selectedLayer.id, { instrument: { sampler: instrument.name, sample: defaultArticulation } })
+        };
+    }
+
+    const onArticulationSelect = async (articulation) => {
+        setSelectedArticulation(articulation.value);
+        // console.log('UPDATE_LAYER_INSTRUMENT', selectedInstrument, event.target.value);
+        dispatch({ type: UPDATE_LAYER_INSTRUMENT, payload: { id: selectedLayer.id, instrument: { sampler: selectedInstrument, sample: articulation.value }, user: user.id } })
+        firebase.updateLayer(roundId, selectedLayer.id, { instrument: { sample: articulation.value } })
+        // if (selectedInstrument === 'custom') {
```
Comment (choshun) [inline src/components/play/layer-settings/LayerInstrumentAlt.js:64]: comment

File: src/components/play/layer-settings/LayerInstrumentAlt.js [hunk 5]
```diff
@@ -0,0 +1,159 @@
+import React, { useContext } from 'react'
+import { Box, Typography } from '@material-ui/core'
+import IconButton from '@material-ui/core/IconButton'
+import _ from 'lodash'
+import { useDispatch } from "react-redux";
+
+import Instruments from '../../../audio-engine/Instruments'
+import { UPDATE_LAYER_INSTRUMENT } from '../../../redux/actionTypes'
+// import CustomSamples from '../../../audio-engine/CustomSamples'
```
Comment (choshun) [inline src/components/play/layer-settings/LayerInstrumentAlt.js:9]: unused import

**Review by choshun — APPROVED** [review]
LG!

File: src/components/play/PlayUI.js (hunk 1 above)
Comment (muganwas) [reply src/components/play/PlayUI.js:51]: I agree but for some reason I got an undefined window error a couple of times on localhost.

File: src/components/play/PlayUI.js (hunk 2 above)
Comment (muganwas) [reply src/components/play/PlayUI.js:102]: Sure.

File: src/components/play/layer-settings/LayerInstrumentAlt.js (hunk 3 above)
Comment (muganwas) [reply src/components/play/layer-settings/LayerInstrumentAlt.js:1]: Ali wanted us to have both the old and new inspector but we'll eventually delete the old files.

**Pull request merged**